### ISSUE TITLE
A possible reason to the superior performace of sphereface2 trained with VGG2

### ISSUE THREAD

**Issue by hiyijian** [ydwen/opensphere#5]
Dear @wy1iu @ydwen 

Thank you for the contribution. 

I noticed that you only trained sphereface2 model with VGG2 dataset in the original paper. As far as I know, VGG2 is a relative dirty dataset which has  quite a lot of wrong intra- and inter-class ID annotation. 

I wonder if the reason to superior performace  is accout for the binary classifier using by sphereface2, because it is not sensitive to wrong annoation as loss function using softmax, such as cosface, arcface and so on. 

what do you think about it?

therefore, I am looking forward to the validation result of more pretrain model trained with relative clean dataset, such as MS1M, Glint360 and Webface42M.  I will be very appreciated if ablation result was shown in this repo :D

**Comment by wy1iu** [comment]
Thanks for the question.   :)

First of all, SphereFace2 is indeed more robust to label noise, as has been tested in the paper. I think this is actually very important in practical face recognition. It could definitely be one of the many reasons that SphereFace2 outperforms the other methods when trained with VGGFace2. 

As for the performance on those relatively clean datasets, we will soon release pretrained models on MS1M (along with the training config), and hopefully Glint360K in the future. We are a bit out of bandwidth due to some recent deadlines, but we are aimming to release it at the end of May. Please stay tuned.

As far as we can observe, SphereFace2 can still achieve very competitive performance on these datasets. One of the weaknesses (or could be an advantage) is that SphereFace has more hyperparameters. Although it will surely make the hyperparameter tuning a bit more painful, it also makes SphereFace2 more flexible and with proper tuning, it will outperform the multi-class classification training methods.

**Comment by hiyijian** [comment]
Thanks for your respond. The issue is going to be closed.
